src/main/java/com/sprint/monew/domain/notification/NotificationRepository.java

```diff
@@ -1,11 +1,22 @@
 package com.sprint.monew.domain.notification;
 
 import com.sprint.monew.domain.user.User;
+
+import java.time.Instant;
 import java.util.List;
 import java.util.UUID;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 
 public interface NotificationRepository extends JpaRepository<Notification, UUID> {
 
   List<Notification> findByUser(User user);
+
+  @Modifying(clearAutomatically = true)
+  @Query("DELETE FROM Notification n " +
+      "where n.confirmed = true " +
+      "and n.updatedAt > :confirmedAt")
+  void deleteConfirmedNotificationsOlderThan(@Param("confirmedAt") Instant confirmedAt);
```

Comment:
```suggestion
  @Modifying(clearAutomatically = true)
  @Query("DELETE FROM Notification n " +
      "where n.confirmed = true " +
      "and n.updatedAt < :confirmedAt")
  void deleteConfirmedNotificationsOlderThan(@Param("confirmedAt") Instant confirmedAt);
```

제가 잘못생각하는 것일수도 있는데 오래된 걸 지우는거라면 부등호가 반대방향이어야하지 않나요?

Reply:
confirmedAt이 현재 시간 기준 7일전의 알림인데, 그것보다 이전의 것들은 제거하는 거라서 현재 부등호가 맞는 것 같습니다